README.md

```diff
@@ -143,7 +141,6 @@ If you want to contribute code then you can fork this repo, make changes and rai
 
 - [Telegram Group](https://t.me/frappebooks): Used for discussions and decisions regarding everything Frappe Books.
 - [GitHub Discussions](https://github.com/frappe/books/discussions): Used for discussions around a specific topic.
-- [Frappe Books Blog](https://tech.frappebooks.com/): Sporadically updated dev blog regarding the development of this project.
```

Comment:
Couldn't find equivalent for https://tech.frappebooks.com/ :confused:

Reply:
Hi @ianandhum, I don't think we have an alternative for that currently.